Snippets/ASB/ASB_7/DeadLetterQueueForwarding.cs

```diff
@@ -0,0 +1,15 @@
+using NServiceBus;
+
+class DeadLetterQueueForwarding
+{
+    void ConfigureDeadLetterQueueForwarding(EndpointConfiguration endpointConfiguration)
+    {
+        #region asb-configure-dead-letter-queue-forwarding
+
+        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
+        transport.Queues().ForwardDeadLetteredMessagesTo(queuename => queuename != "error" && queuename != "audit" && queuename != "centralizeddlq", "centralizeddlq");
```

Comment:
What's the purpose of a complex check here for the DLQ assignment? Why not a simple queue assignment?